pjsip-apps/src/swig/java/android/app/src/main/java/org/pjsip/pjsua2/app/MyApp.java

```diff
@@ -373,6 +408,9 @@ public void init(MyAppObserver obs, String app_dir,
             System.out.println(e);
         }
 
+        // TCP & TLS transports with default SIP ports
+        // sometimes not working fine on Android somehow.
```

Comment:
It works fine on my device, so I have to uncomment it here.
Perhaps you can try using non-default ports rather than disabling this entirely?